bulletroboy/operator.py

```diff
@@ -192,47 +248,56 @@ def two_end_effectors(self, positions, maxIter = 100, chest_constraint = True):
 	def simple_move(self, case):
```

Comment:
I think this method is too big, is there a way to split it?

Reply:
It's already smaller in the movement-class-fix branch, this PR is only about the human urdf file

Reply:
so i dont want to / not planning to review the same code multiple times. if you have a new version, where is it?